stake-pool/js/package.json

```diff
@@ -1,21 +1,40 @@
 {
   "name": "@solana/spl-stake-pool",
-  "version": "0.2.1",
+  "version": "0.6.3",
   "description": "SPL Stake Pool Program JS API",
   "scripts": {
-    "lint": "npx prettier src/*.ts -w",
+    "build": "tsc",
```

Comment:
This used to be `compile` instead of `build`, documentation should probably be updated to reflect this change.

Reply:
Excellent, great find! I'll fix this up right away